sdk/identity/identity/src/credentials/defaultAzureCredential.ts

```diff
@@ -260,10 +129,33 @@ export class DefaultAzureCredential extends ChainedTokenCredential {
           // If AZURE_TOKEN_CREDENTIALS is set to "prod", use the production credential chain.
           credentialFunctions = prodCredentialFunctions;
           break;
+        case "environmentcredential":
```

Comment:
Is this case sensitive? I ask because the documentation uses case-sensitive terms, but here everything is in lowercase.

Reply:
No it is case-insensitive. I think in documentation, it is more readable that way.